app/upgrades_test.go

```diff
@@ -94,3 +98,64 @@ func (suite *UpgradeTestSuite) TestUpdateSavingsParams() {
 		"SupportedDenoms should be updated to include ukava",
 	)
 }
+
+func (suite *UpgradeTestSuite) TestConvertEOAsToBaseAccount() {
+	ak := suite.App.GetAccountKeeper()
+
+	accCount := 0
+
+	// Add all accounts as EthAccount
+	app.IterateEOAAddresses(func(addr string) {
```

Comment:
this test is running through all 33k records right? How long does this take?

Reply:
Yep it goes through them all, the test consistently finishes in under 1s for me